flang/test/Driver/omp-driver-offload.f90

```diff
@@ -137,7 +137,7 @@
 ! RUN: -fopenmp-targets=nvptx64-nvidia-cuda \
 ! RUN: -fopenmp-target-debug \
 ! RUN: | FileCheck %s --check-prefixes=CHECK-TARGET-DEBUG
-! CHECK-TARGET-DEBUG-EQ: "{{[^"]*}}flang-new" "-fc1" {{.*}} "-fopenmp" {{.*}} "-fopenmp-is-target-device" "-fopenmp-target-debug=111" {{.*}}.f90"
+! CHECK-TARGET-DEBUG: "{{[^"]*}}flang-new" "-fc1" {{.*}} "-fopenmp" {{.*}} "-fopenmp-is-target-device" "-fopenmp-target-debug=111" {{.*}}.f90"
```

Comment:
The fix here looks right but the test fails: flang uses `-fopenmp-target-debug` without the `=111`. @skatrak is this correct?

Reply:
I can see I was the last one to modify that line, but it was unrelated (the "-EQ") was there before. I think @agozillon might know a bit more about this, but that check indeed fails for me as well if enabled.

Reply:
Looks like the test was broken by a typo originally (my appologies), the `--check-prefixes` line should originally have been: `--check-prefixes=CHECK-TARGET-DEBUG-EQ` to seperate it from the test prior to it that utilises CHECK-TARGET-DEBUG. And the actual -fopenmp-target-debug  command argument should be changed to "-fopenmp-target-debug=111" for both the CUDA and AMD variations of the command, that should allow it to pass.